client/state/account-recovery/reset/test/reducer.js

```diff
@@ -16,6 +16,17 @@ import {
 import reducer from '../reducer';
 
 describe( '#account-recovery/reset reducer', () => {
+	const fetchedOptions = [
```

Comment:
Good idea :) 
Added in several similar places in e337232ac428f5caec617ad82dd2b382a000dbe1.